packages/recommend-js/src/utils/withHtml.tsx

```diff
@@ -0,0 +1,17 @@
+/** @jsx h */
+import { html } from 'htm/preact';
+import { h } from 'preact';
+
+import { HTMLTemplate } from '../types';
+
+export function withHtml<TProps>(
+  Component: preact.FunctionComponent<TProps>
+): (props: TProps & HTMLTemplate) => JSX.Element {
+  const ComponentWithHtml = (props: TProps) => (
+    <Component {...props} html={html} />
+  );
+
+  ComponentWithHtml.displayName = `withHtml(${Component.displayName})`;
```

Comment:
We can save some bytes on the production bundle:

```suggestion
  if (__DEV__) {
    ComponentWithHtml.displayName = `withHtml(${Component.displayName})`;
  }
```